torchx/mix.exs

```diff
@@ -43,4 +45,119 @@ defmodule Torchx.MixProject do
       source_url: @source_url
     ]
   end
+
+  defp compilers do
+    if System.get_env("LIBTORCH_DIR"), do: [:elixir_make], else: [:torchx, :elixir_make]
+  end
+
+  defp aliases do
+    [
+      "compile.torchx": &compile/1
+    ]
+  end
+
+  defp compile(args) do
+    cache_dir = Path.join(__DIR__, "cache")
+    # TODO: This directory name must also include the libtorch version.
+    # Otherwise if someone depends on torchx as a git dependency, they
+    # can upgrade the version and we will think it is cached but it is actually old.
+    libtorch_dir = Path.join(cache_dir, "libtorch_#{@default_libtorch_version}")
+
+    if "--force" in args do
+      File.rm_rf!(cache_dir)
```

Comment:
```suggestion
      File.rm_rf!(libtorch_dir)
```

Reply:
I think we want to remove cache dir itself. Otherwise libtorch won't ever be downloaded again without manually erasing cache/libtorch.zip

Reply:
Good call!